ducktape/tests/result.py

```diff
@@ -48,7 +48,7 @@ def __init__(self,
             self.nodes_used = test_context.services.num_nodes()
         else:
             self.services = {}
-            self.nodes_used = 0
+            self.nodes_used = {"": 0}
```

Comment:
Well, when I use `nodes_used` below, I ignore the key. So the key doesn't matter. However, I realize that's confusing, so I'll make the key LINUX, which is the default elsewhere. That will be less confusing.